actions/raw_action.go

```diff
@@ -1,3 +1,28 @@
+/*
+Raw Action
+
+Allow to write some file to output image with given offset.
```

Comment:
I’d drop `Allow to`. Or replace with `Allows to`. Subject to your taste ;)

Reply:
done